File: Búsqueda/Secuencial/secuencial.java
public class secuencial {

    public static int busquedaSecuencial(int[] arreglo, int elemento) {
        for (int i = 0; i < arreglo.length; i++) {
            if (arreglo[i] == elemento) {
                return i; // Retorna el índice donde se encuentra el elemento
            }
        }
        return -1; // Retorna -1 si el elemento no se encuentra en el arreglo
    }

    public static void main(String[] args) {
        int[] arreglo = {5, 3, 7, 1, 4, 9, 2};
        int elemento = 4;
        int resultado = busquedaSecuencial(arreglo, elemento);

        if (resultado != -1) {
            System.out.println("Elemento encontrado en el índice: " + resultado);
        } else {
            System.out.println("Elemento no encontrado en el arreglo.");
        }
    }
}

File: Búsqueda/Secuencial/secuencial.py
def busqueda_secuencial(lista, objetivo):
    """
    Realiza una búsqueda secuencial en una lista.

    Parámetros:
    lista (list): La lista en la que se va a buscar.
    objetivo: El elemento que se desea encontrar.

    Retorna:
    int: El índice del elemento si se encuentra, de lo contrario -1.
    """
    for indice, elemento in enumerate(lista):
        if elemento == objetivo:
            return indice
    return -1

# Ejemplo de uso
lista = [3, 5, 2, 4, 9]
objetivo = 4
resultado = busqueda_secuencial(lista, objetivo)
if resultado != -1:
    print(f'Elemento encontrado en el índice: {resultado}')
else:
    print('Elemento no encontrado')